### ISSUE TITLE
Page title "Your recording is ready!" incomplete

### ISSUE THREAD

**Issue by cyberbaloo** [numerique-gouv/meet#1091]
## Bug Report

**Problematic behavior**
The HTML page title is "Visio". The user cannot immediately identify that they are on the meeting recording page.

**Expected behavior/code**
Make the page title explicit in the HTML  <title> 

**Steps to Reproduce**
1. Open a meeting and start the recording, then close it
2. Check your emails and click on open
3. And then the bug happens!

**Additional context/Screenshots**

<img width="215" height="31" alt="Image" src="https://github.com/user-attachments/assets/eca8952d-8329-4cab-b59b-25520a5312db" />
